File: app/src/main/java/com/yj/banner/xxxbanner/loader/GlideImageLoader.java
package com.yj.banner.xxxbanner.loader;

import android.content.Context;
import android.widget.ImageView;

import com.yj.banner.xxxbanner.util.GlideUtil;
import com.yj.xxxbanner.loader.ImageLoaderInterface;


public class GlideImageLoader implements ImageLoaderInterface<ImageView, String> {
    @Override
    public void displayView(Context context, String path, ImageView view) {
        GlideUtil.INSTANCE.loadPicture(view, path);
    }

    @Override
    public ImageView createView(Context context) {
        return new ImageView(context);
    }
}

File: app/src/main/java/com/yj/banner/xxxbanner/loader/GlideImageLoader.kt
package com.yj.banner.xxxbanner.loader

import android.content.Context
import android.widget.ImageView

import com.yj.banner.xxxbanner.util.GlideUtil
import com.yj.xxxbanner.loader.LoaderInterface


class GlideImageLoader : LoaderInterface<ImageView, String> {
    override fun displayView(context: Context, bean: String, view: ImageView, position: Int, count: Int) {
        GlideUtil.loadPicture(view, bean)
    }



    override fun createView(context: Context): ImageView {
        return ImageView(context)
    }
}
